src/config/config.js

```diff
@@ -0,0 +1,18 @@
+export const defaultConfig = {
+  timeOffsets: {
+    passiveRenewal: 300000,
+  },
+  onRenewSuccess: (sessionExpiryTime, refreshExpiryTime) => console.debug(
+    `[LIBRARY] Session renewed successfully. Session: ${sessionExpiryTime} and refresh: ${refreshExpiryTime}`,
+  ),
+  onRenewFailure: () => console.warn('[LIBRARY] Session renewal failed'),
+  onSessionValid: (sessionExpiryTime, refreshExpiryTime) => console.debug(
+    `[LIBRARY] Session Valid. Session: ${sessionExpiryTime} and refresh: ${refreshExpiryTime}`,
+  ),
+  onSessionInvalid: () => console.warn('[LIBRARY] Session is invalid'),
+  onError: (error) => console.error('[LIBRARY] Error:', error),
+};
+
+export const apiConfig = {
+  RENEW_SESSION: '/tokens/self',
```

Comment:
```suggestion
  RENEW_SESSION: '/api/v1/tokens/self',
```
I *think* this should just work - but try it all the same.